### DISCUSSION TITLE
Flutter - Timeout preparing builder machine, please restart the build

### DISCUSSION THREAD

**Discussion by MewanWA** [codemagic-ci-cd/codemagic-docs#2803] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1. Run codemagic build (iOS or Android)

### Expected results

1. The build should run as usual without timing out.

### Actual results

1. Run codemagic build (iOS or Android)
2. Preparing Build Machine step timesout after 10 mins
3. No logs are shown on why it timed out.

### Build id (optional)

67208ae38a327985eccf6c08

**Comment by dam-ease** [comment]
Hello @MewanWA,
We have an ongoing thread on Discord currently, please let's resolve the issue there while I close this in favour of the discord thread.
